client/components/user/auth/login/login.template.html

```diff
@@ -1,23 +1,30 @@
-<div class="container login">
+<div class="registration container">
```

Comment:
Swap `container` and `registration`. This will prevent _Bootstrap_ from accidentally overriding your styles.